libcflib/analyze_pkgs.py

```diff
@@ -11,6 +11,12 @@
 from concurrent.futures import as_completed, ThreadPoolExecutor
 from itertools import groupby
 
+CLOBBER_EXCEPTIONS = {
+    'matplotlib',
+    'matplotlib-base',
+    'mongo',
+}
```

Comment:
I'd prefer us to not make exceptions here and instead list everything that could clobber.

Reply:
Those exceptions only apply to the list of pkgs who clobber, not the file listing. My plan is to use the list of clobbering pkgs for things like depfinder, since the file listing might be a bit big and have many negative results. Is that ok?

Reply:
OK. As long as everything is in the file listing that is great.

Reply:
the only usage is here: https://github.com/regro/libcflib/pull/58/files#diff-5977cf48a353720d200a6af738fcbb4ab55b308ca0ff907c02b2c77fb191d1fbR115